sockeye/inference.py

```diff
@@ -787,10 +867,10 @@ def _beam_search(self,
                      source: mx.nd.NDArray,
                      source_length: int) -> Tuple[mx.nd.NDArray, mx.nd.NDArray, mx.nd.NDArray, mx.nd.NDArray]:
         """
-        Translates a single sentence using beam search.
+        Translates multiple sentences using beam search.
 
-        :param source: Source ids. Shape: (1, bucket_key).
-        :param source_length: Source length.
+        :param source: Source ids. Shape: (num_sent, bucket_key). Num_sent can be from 1 up to self.batch_size
```

Comment:
why is it a range? isnt it always exactly batch_size (which can be 1)?

Also missing period.